modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -148,12 +156,127 @@ spec:
       - _tag in Exchange asset_ # example: orders
       apiInstance:
       - _tag in API Manager instance_ # example: orders
-  parametersFrom: # The element specifies the secret name and configuration required for retrieving the parameters.
+    parametersFrom: # The element specifies the secret name and configuration required for retrieving the parameters.
     - secretKeyRef:
         name: _secret name_ # example: orders-api-config
         key: config
```

Comment:
The position of `parametersFrom` was the correct one, the change to align it with `secretKeyRef` would be:
```suggestion
  parametersFrom: # The element specifies the secret name and configuration required for retrieving the parameters.
  - secretKeyRef:
      name: _secret name_ # example: orders-api-config
      key: config
```
Would you mind performing the same change in: https://github.com/mulesoft/docs-service-mesh/blame/v1.1/modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc#L276

Reply:
Fixed, check with Fran!

Reply:
What i'm saying is that `parametersFrom` has to be sibling of `parameters`, not a child of it. It other words, both have to be at the same level.

Both examples, creating an API via CRD with user and connected apps credentials, have the same mistake, `parametersFrom` is placed as if it were a child of `parameters`.